### PR TITLE
[#5] Add FE validation logic; Update readme and examples;

### PR DESCRIPTION
**Pull request by svetozarsadecky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,10 +111,14 @@ export default {
 | `readonly` | boolean | `false` | If field is read only |
 | `placeholder` | string | `` | Native placeholder attribute for input/textarea |
 | `accept` | string |  | Which file types should be accepted if type is file |
-| `validators` | array | `[]` | Which validators should be used to validate input value |
+| `validators` | array | `undefined` | Array holding objects with `validator`{`string`} and `message`{`string`} properties |
```
Comment (odyzeo) [inline README.md]: ked je array default daj `[]` alebo ked tak `null` ale `undefined` not cool

File: README.md [hunk 2]
```diff
@@ -132,13 +136,45 @@ Error message for required field
 - `min` : length of string
 - `confirmed `: confirmed passwords
 - `regex` : your custom regex
+- `required` : for custom required message
```
Comment (odyzeo) [inline README.md:143]: mozno by sme modli pridat aj nejaky custom validate funkciu aby si mohol sam vlozit nejaky validator nieco ako:

https://vuelidate.netlify.com/#sub-simplest-example

File: src/components/FormItem.vue [hunk 3]
```diff
@@ -9,10 +9,12 @@
     >
         <div class="form-item__field">
             <slot name="prepend"></slot>
+
```
Comment (odyzeo) [inline src/components/FormItem.vue:12]: musime sa porozpravat o tych tvojich prazdnych riadkoch :D

File: src/components/FormItem.vue [hunk 4]
```diff
@@ -87,21 +97,25 @@ import VALIDATORS from '@/constants/validators';
...
         formErrors: {
             type: [Array, Object],
-            default: () => [],
+            default: () => ([]),
         },
         value: {
             type: String,
             default: '',
         },
         msgRequired: {
```
Comment (odyzeo) [inline src/components/FormItem.vue]: toto by som vyhodil, zbytocny props budeme to definovat vo validators
ak by vadila kompatibilita tak publishneme major verziu

File: src/components/FormItem.vue [hunk 5]
```diff
@@ -124,7 +138,8 @@ export default {
             errors: [],
             validating: false,
             isTextArea: type === 'textarea',
-            showFormErrors: !!this.formErrors,
+            showFormErrors: (this.formErrors.length > 0),
```
Comment (odyzeo) [inline src/components/FormItem.vue]: velmi zle co ak je formErrors undefined ? - padne mi to na `Cannot read property 'length' of undefined`

File: src/components/FormItem.vue (hunk 4 above)
Comment (svetozarsadecky) [reply src/components/FormItem.vue]: Resolved in 41869c9

File: src/components/FormItem.vue (hunk 3 above)
Comment (svetozarsadecky) [reply src/components/FormItem.vue:12]: Resolved in 41869c9

File: README.md (hunk 2 above)
Comment (svetozarsadecky) [reply README.md:143]: Added in 41869c9

File: README.md (hunk 1 above)
Comment (svetozarsadecky) [reply README.md]: Resolved in 41869c9

File: src/plugin/formItem.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+         * Makes sure that plugin can be installed only once
+         */
+        if (this.installed) {
+            return;
+        }
+
+        /**
+         * Functions
+         */
+        function getActiveItems(item) {
```
Comment (LordZombi) [inline src/plugin/formItem.js]: uz som ti vela krat vravel NAMING! ak sa vola funkcia `get...` a ty mi nic nevratis zomrie jedno maciatko

**Pull request merged**